src/main/java/team/themoment/hellogsmv3/domain/application/service/ModifyApplicationService.java

```diff
@@ -0,0 +1,269 @@
+
+    private final ApplicantRepository applicantRepository;
+    private final AuthenticationRepository authenticationRepository;
+    private final ApplicationRepository applicationRepository;
+    private final EntityManager em;
+
+    @Transactional
+    public void execute(ApplicationReqDto dto, Long userId, boolean isAdmin) {
+
+        Applicant applicant = applicantRepository.findById(userId)
+                .orElseThrow(() -> new ExpectedException("존재하지 않는 유저입니다.", HttpStatus.NOT_FOUND));
+
+        AbstractApplication application = applicationRepository.findByApplicant(applicant)
+                .orElseThrow(() -> new ExpectedException("원서를 찾을 수 없습니다.", HttpStatus.NOT_FOUND));
+
+        if (!isAdmin && application.getFinalSubmitted())
+            throw new ExpectedException("최종제출이 완료된 원서는 수정할 수 없습니다.", HttpStatus.BAD_REQUEST);
+
+        if (!authenticationRepository.existsById(applicant.getAuthenticationId()))
+            throw new ExpectedException("인증 정보가 존재하지 않습니다.", HttpStatus.NOT_FOUND);
+
+        applicationRepository.delete(application);
+        em.flush();
+
+        if (dto.graduation() == GraduationStatus.CANDIDATE) {
+
+            CandidatePersonalInformation candidatePersonalInformation = createCandidatePersonalInformation(dto, application);
+
+            CandidateMiddleSchoolGrade candidateMiddleSchoolGrade = createCandidateMiddleSchoolGrade(application);
+
+            CandidateApplication candidateApplication = createCandidateApplication(
+                    dto, candidatePersonalInformation, candidateMiddleSchoolGrade, applicant, application);
+
+            applicationRepository.save(candidateApplication);
+
+        } else if (dto.graduation() == GraduationStatus.GRADUATE) {
+
+            GraduatePersonalInformation graduatePersonalInformation = createGraduatePersonalInformation(dto, application);
+
+            GraduateMiddleSchoolGrade graduateMiddleSchoolGrade = createGraduateMiddleSchoolGrade(application);
+
+            GraduateApplication graduateApplication = createGraduateApplication(
+                    dto, graduatePersonalInformation, graduateMiddleSchoolGrade, applicant, application);
+
+            applicationRepository.save(graduateApplication);
+
+        } else if (dto.graduation() == GraduationStatus.GED) {
+
+            GedPersonalInformation gedPersonalInformation = createGedPersonalInformation(dto, application);
+
+            GedMiddleSchoolGrade gedMiddleSchoolGrade = createGedMiddleSchoolGrade(application);
+
+            GedApplication gedApplication = createGedApplication(dto, gedPersonalInformation, gedMiddleSchoolGrade, applicant, application);
+
+            applicationRepository.save(gedApplication);
+
+        } else {
+            throw new RuntimeException("발생하면 안되는 에러, 존재하지 않는 졸업현황");
+        }
+
+    }
+
+    private CandidatePersonalInformation createCandidatePersonalInformation(ApplicationReqDto dto, AbstractApplication application) {
```

Comment:
```suggestion
    private CandidatePersonalInformation buildCandidatePersonalInformation(ApplicationReqDto dto, AbstractApplication application) {
```

create 는 뭔가 DB save 작업을 해서 실제로 data가 생성을 하는 operation method로 많이 쓰이는 prefix라서.
단순 객체 생성이라면 `build` prefix 를 추천드려봅니다.

Reply:
아래에 딸려오는 `create**` 로 정의된 `build` 형 메소드에도 마찬가지 제안을 드립니다..

Reply:
1b52ccd61172bd1b121772ba4132e71ddf323154

원서 객체를 단순 생성하는 메서드들의 네이밍을 `build**`로 변경하였습니다.